test/es-module/test-esm-exports.mjs

```diff
@@ -173,7 +176,7 @@ function assertIncludes(actual, expected) {
     '/node_modules/pkgexports/resolve-self.js',
     'Package name self resolution',
   ],
-].forEach(([flag, file, message]) => {
+].forEach(([flag, file, message], index) => {
```

Comment:
`index` is added here but it doesn't look like it's used?

Reply:
Thanks, I've removed this.